client/blocks/inline-help/popover.jsx

```diff
@@ -353,6 +375,30 @@ const optIn = ( siteId, gutenbergUrl ) => {
 	);
 };
 
+const upworkNudgeViewed = () => {
+	return composeAnalytics(
+		recordGoogleEvent(
+			'Upwork Link Viewed',
+			'Viewed "Need a designer to build your site?" in the help popover.',
+			'View',
+			false
+		),
+		recordTracksEvent( 'calypso_upwork_help_popover_view' )
+	);
+};
+
+const upworkNudgeClicked = () => {
+	return composeAnalytics(
```

Comment:
these 2 analytics functions could drop the curlies and `return` i.e.
`const upworkNudgeClicked = () => composeAnalytics(`